src/pages/workspace/workflows/ToggleSettingsOptionRow.tsx

```diff
@@ -9,12 +10,18 @@ import type {Errors, PendingAction} from '@src/types/onyx/OnyxCommon';
 import type IconAsset from '@src/types/utils/IconAsset';
 
 type ToggleSettingOptionRowProps = {
+    // Key used internally by React
+    key?: string;
```

Comment:
```suggestion
```
Not needed. `key` can be passed without being explicitly defined. Also this is causing a console error
<img width="789" alt="Screenshot 2024-04-18 at 1 40 31 AM" src="https://github.com/Expensify/App/assets/16493223/bd78f65b-6924-40b5-b8de-377a7b0b928c">